src/main/webapp/lectures/index.html

```diff
@@ -58,7 +58,16 @@ <h3 class="text-center">Add a Lecture</h3>
           <form name="lectureForm" action="/lecture" method="post">
             <input type="text" class="form-control" placeholder="Lecture Name" name="name-input" required />
             <br />
-            <input type="url" class="form-control" placeholder="Lecture YouTube URL" name="link-input" required />
+            <input
+              type="url"
+              class="form-control"
+              placeholder="Lecture YouTube URL"
+              name="link-input"
+              oninput="fetchAndDisplayTranscriptLanguages(this)"
+              required
+            />
```

Comment:
Nit: I would move this to the previous line.

Reply:
I think the linter does this automatically, should I still fix this?

Reply:
If the linter does this, then keep it as is.